packages/indexer-common/src/network.ts

```diff
@@ -244,17 +244,18 @@ export class Network {
     // --------------------------------------------------------------------------------
     const networkIdentifier = await networkProvider.getNetwork()
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    let tapContracts: any
-    try {
-      tapContracts = await connectTapContracts(
-        wallet,
-        networkIdentifier.chainId,
-        specification.tapAddressBook,
-      )
-    } catch (err) {
-      logger.error(`Failed to connect to tap contract bindings:`, { err })
+    let tapContracts: any = undefined
```

Comment:
We do know what type this would be if not undefined though, right? 

So we can do:
`let tapContracts: TapContracts | undefined = undefined`